tests/geospatial/test_rechunking.py

```diff
@@ -0,0 +1,26 @@
+import dask
+import pytest
+import xarray as xr
+from coiled.credentials.google import CoiledShippedCredentials
+
+
+@pytest.mark.client("era5_rechunking")
+def test_era5_rechunking(client, gcs_url):
+    # Load dataset
+    ds = xr.open_zarr(
+        "gs://weatherbench2/datasets/era5/1959-2023_01_10-full_37-1h-0p25deg-chunk-1.zarr",
+    ).drop_encoding()
+
+    time_range = slice("2020-01-01", "2023-01-01")  # 101.83 GiB (small)
```

Comment:
Currently just hard-coding this one. Here's the cluster for when this is running https://cloud.coiled.io/clusters/570092/account/dask-engineering-gcp/information?workspace=dask-engineering-gcp&tab=Metrics.